connectors/blue-prism/element-templates/blue-prism-connector.json

```diff
@@ -0,0 +1,462 @@
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "oauth-client-credentials-flow"
+      }
+    },
+    {
+      "id": "methodPost",
+      "group": "configuration",
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createItemInQueue"
+        ]
+      }
+    },
+    {
+      "id": "methodGet",
+      "group": "configuration",
+      "type": "Hidden",
+      "value": "get",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getItemFromQueueById"
+        ]
+      }
+    },
+    {
+      "label": "Blue Prism API base URL",
+      "group": "configuration",
+      "type": "String",
+      "value": "https://blueprism.myservice.com",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "bluePrismBaseUrl"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Work Queue ID",
+      "group": "input",
+      "description": "The queue ID of the item",
+      "type": "String",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "bluePrismQueueId"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Queue item ID",
+      "description": "The queue item identifier to be fetched from queue",
+      "group": "input",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "bluePrismItemId"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getItemFromQueueById"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "Data",
+      "description": "Represents a Blue Prism collection.",
+      "group": "input",
+      "type": "String",
+      "feel": "required",
+      "value": "= { rows: [{\"yourKey\":\"yourValue\"}] }",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "bluePrismData"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createItemInQueue"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "Defer date",
+      "description": "The earliest time and date that this item is deferred until. Expected date format: yyyy-MM-dd",
+      "group": "input",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "deferDate"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createItemInQueue"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "Priority",
+      "description": "[FIXME - integer] The priority value assigned to the item",
```

Comment:
i guess, this is a placeholder for a non existing feature?

Reply:
Yes, fixed now